Gemfile

```diff
@@ -7,8 +7,8 @@ gem 'coffee-rails', '~> 4.0.0'
 gem 'jbuilder', '~> 1.2'
 gem 'jquery-rails', '~> 3.1.0'
 gem 'jwt' # For Zendesk SSO
-gem 'rails_config', '~> 0.3.3'
-gem 'rb-readline', '~> 0.4.2', require: false
+gem 'rails_config', '~> 0.4.2'
```

Comment:
Because I wanted to update Gemfile.lock to use a newer version.  Running this under Ruby 2.1.5 failed with various errors when using the old versions.